specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json

```diff
@@ -504,6 +504,14 @@
         "ImmediatePurgeDataOn30Days": {
           "type": "boolean",
           "description": "Purge data immediately after 30 days."
+        },
+        "publicNetworkAccessForIngestion": {
```

Comment:
>publicNetworkAccessForIngestion" [](start = 9, length = 32)

Needs a new api-version.  See comments on #8808

Reply:
Hi Chris,
This is a nullable enum, in other words, if property is not set we take the existing one. So in the case you mentioned, if user sets Disabled, and then user doesn't set, it will still be Disabled.
If there's other way we should set this in the swagger, please let me know.
Thanks, Arik.